requirements/edx-sandbox/base.txt

```diff
@@ -4,10 +4,10 @@
 #
 #    make upgrade
 #
-common/lib/calc
-common/lib/chem
-common/lib/sandbox-packages
-common/lib/symmath
+-e file:///home/cliff/work/opencraft/docker/edx-platform/common/lib/calc
+-e file:///home/cliff/work/opencraft/docker/edx-platform/common/lib/chem
+-e file:///home/cliff/work/opencraft/docker/edx-platform/common/lib/sandbox-packages
+-e file:///home/cliff/work/opencraft/docker/edx-platform/common/lib/symmath
```

Comment:
@jmbowman Do you know what would cause `make upgrade` to create artifacts like this?

Reply:
Hitting Ctrl-C before the make target finishes would do it; that gets cleaned up in the post-processing script: https://github.com/edx/edx-platform/blob/master/scripts/post-pip-compile.sh#L22